tests/js/spec/views/issueList/searchBar.spec.jsx

```diff
@@ -258,7 +258,7 @@ describe('IssueListSearchBar', function () {
         organization,
       };
       const searchBar = mountWithTheme(<IssueListSearchBar {...props} />, routerContext);
-      expect(searchBar.find('[data-test-id="pin-icon"]')).toHaveLength(2);
+      expect(searchBar.find('[data-test-id="pin-icon"]')).toHaveLength(3);
```

Comment:
was it wrong?

Reply:
This change is caused by adding `withTheme` to the definition of DropdownElement.